src/client/components/localisation/view_model.ts

```diff
@@ -21,7 +22,11 @@ export class LocalisationViewModel {
   public get scene(): Scene {
     const scene = new Scene()
     this.robots.forEach(robot => scene.add(robot))
+
     scene.add(this.field)
+    scene.add(this.skybox)
+    scene.add(this.skyboxGround)
+    scene.add(this.skyboxSun)
```

Comment:
I think you should combine these 3 into a single `this.skybox`, so that adding the skybox is a single scene addition e.g.

Usage: `scene.add(this.skybox)`

Impl:
```
@computed
private get skybox() {
   const skybox = new Object3D()
   skybox.add(this.skyboxSky) // <-- this is currently called skybox
   skybox.add(this.skyboxGround)
   skybox.add(this.skyboxSun)
   return skybox
}
```